src/CommunityToolkit.Aspire.Hosting.PapercutSmtp/PapercutSmtpHostingExtension.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Aspire.Hosting.ApplicationModel;
+using CommunityToolkit.Aspire.Hosting.Ollama;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding DataApiBuilder api to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class PapercutSmtpHostingExtension
+{
+    /// <summary>
+    /// Adds Papercut SMTP to the application model.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/> to add the resource to.</param>
+    /// <param name="name">The name of the resource.</param>
+    /// <remarks>
+    /// </remarks>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<PapercutSmtpContainerResource> AddPapercutSmtp(this IDistributedApplicationBuilder builder,
+        [ResourceName] string name)
+    {
+        return builder.AddPapercutSmtp(name, null);
+    }
```

Comment:
This can be removed as the other method provides defaults for `httpPort` and `smtpPort` so you can call it without any argument other than `name`.